[
   {
      "body" : "According to the message (translated from italian) this is an I/O error. This likely means there is a problem with the harddisk.",
      "created_at" : "2016-11-09T21:23:36Z",
      "html_url" : "https://github.com/bitcoin/bitcoin/issues/9119#issuecomment-259530594",
      "id" : 259530594,
      "issue_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/9119",
      "updated_at" : "2016-11-09T21:23:36Z",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/259530594",
      "user" : {
         "avatar_url" : "https://avatars.githubusercontent.com/u/126646?v=3",
         "events_url" : "https://api.github.com/users/laanwj/events{/privacy}",
         "followers_url" : "https://api.github.com/users/laanwj/followers",
         "following_url" : "https://api.github.com/users/laanwj/following{/other_user}",
         "gists_url" : "https://api.github.com/users/laanwj/gists{/gist_id}",
         "gravatar_id" : "",
         "html_url" : "https://github.com/laanwj",
         "id" : 126646,
         "login" : "laanwj",
         "organizations_url" : "https://api.github.com/users/laanwj/orgs",
         "received_events_url" : "https://api.github.com/users/laanwj/received_events",
         "repos_url" : "https://api.github.com/users/laanwj/repos",
         "site_admin" : false,
         "starred_url" : "https://api.github.com/users/laanwj/starred{/owner}{/repo}",
         "subscriptions_url" : "https://api.github.com/users/laanwj/subscriptions",
         "type" : "User",
         "url" : "https://api.github.com/users/laanwj"
      }
   },
   {
      "body" : "thanks for your  quick answer, what should I do now? do I have to check the  hard disk or something else?",
      "created_at" : "2016-11-09T21:29:26Z",
      "html_url" : "https://github.com/bitcoin/bitcoin/issues/9119#issuecomment-259531950",
      "id" : 259531950,
      "issue_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/9119",
      "updated_at" : "2016-11-09T21:29:26Z",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/259531950",
      "user" : {
         "avatar_url" : "https://avatars.githubusercontent.com/u/23367015?v=3",
         "events_url" : "https://api.github.com/users/Bitoma/events{/privacy}",
         "followers_url" : "https://api.github.com/users/Bitoma/followers",
         "following_url" : "https://api.github.com/users/Bitoma/following{/other_user}",
         "gists_url" : "https://api.github.com/users/Bitoma/gists{/gist_id}",
         "gravatar_id" : "",
         "html_url" : "https://github.com/Bitoma",
         "id" : 23367015,
         "login" : "Bitoma",
         "organizations_url" : "https://api.github.com/users/Bitoma/orgs",
         "received_events_url" : "https://api.github.com/users/Bitoma/received_events",
         "repos_url" : "https://api.github.com/users/Bitoma/repos",
         "site_admin" : false,
         "starred_url" : "https://api.github.com/users/Bitoma/starred{/owner}{/repo}",
         "subscriptions_url" : "https://api.github.com/users/Bitoma/subscriptions",
         "type" : "User",
         "url" : "https://api.github.com/users/Bitoma"
      }
   },
   {
      "body" : "Yes, you could try running some disk checks/diagnostics. If the disk is broken the only safe course of action would be to replace it. ",
      "created_at" : "2016-11-09T21:38:50Z",
      "html_url" : "https://github.com/bitcoin/bitcoin/issues/9119#issuecomment-259533965",
      "id" : 259533965,
      "issue_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/9119",
      "updated_at" : "2016-11-09T21:38:50Z",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/259533965",
      "user" : {
         "avatar_url" : "https://avatars.githubusercontent.com/u/126646?v=3",
         "events_url" : "https://api.github.com/users/laanwj/events{/privacy}",
         "followers_url" : "https://api.github.com/users/laanwj/followers",
         "following_url" : "https://api.github.com/users/laanwj/following{/other_user}",
         "gists_url" : "https://api.github.com/users/laanwj/gists{/gist_id}",
         "gravatar_id" : "",
         "html_url" : "https://github.com/laanwj",
         "id" : 126646,
         "login" : "laanwj",
         "organizations_url" : "https://api.github.com/users/laanwj/orgs",
         "received_events_url" : "https://api.github.com/users/laanwj/received_events",
         "repos_url" : "https://api.github.com/users/laanwj/repos",
         "site_admin" : false,
         "starred_url" : "https://api.github.com/users/laanwj/starred{/owner}{/repo}",
         "subscriptions_url" : "https://api.github.com/users/laanwj/subscriptions",
         "type" : "User",
         "url" : "https://api.github.com/users/laanwj"
      }
   },
   {
      "body" : "ok ....I try.\r\n\r\nThanks for your help",
      "created_at" : "2016-11-09T21:39:51Z",
      "html_url" : "https://github.com/bitcoin/bitcoin/issues/9119#issuecomment-259534202",
      "id" : 259534202,
      "issue_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/9119",
      "updated_at" : "2016-11-09T21:39:51Z",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/259534202",
      "user" : {
         "avatar_url" : "https://avatars.githubusercontent.com/u/23367015?v=3",
         "events_url" : "https://api.github.com/users/Bitoma/events{/privacy}",
         "followers_url" : "https://api.github.com/users/Bitoma/followers",
         "following_url" : "https://api.github.com/users/Bitoma/following{/other_user}",
         "gists_url" : "https://api.github.com/users/Bitoma/gists{/gist_id}",
         "gravatar_id" : "",
         "html_url" : "https://github.com/Bitoma",
         "id" : 23367015,
         "login" : "Bitoma",
         "organizations_url" : "https://api.github.com/users/Bitoma/orgs",
         "received_events_url" : "https://api.github.com/users/Bitoma/received_events",
         "repos_url" : "https://api.github.com/users/Bitoma/repos",
         "site_admin" : false,
         "starred_url" : "https://api.github.com/users/Bitoma/starred{/owner}{/repo}",
         "subscriptions_url" : "https://api.github.com/users/Bitoma/subscriptions",
         "type" : "User",
         "url" : "https://api.github.com/users/Bitoma"
      }
   },
   {
      "body" : "I run a disk checks/diagnostics.  Everything seems ok.  But, when bitcoin core start again , after a while it stops. the  log debug writes:\r\n2016-11-11 14:15:00 IO error: C:\\Users\\PC\\AppData\\Roaming\\Bitcoin\\chainstate\\304129.ldb: Impossibile eseguire la \r\nrichiesta a causa di un errore di dispositivo I/O.\r\n2016-11-11 14:15:00 *** System error while flushing: Database I/O error\r\n2016-11-11 14:15:09 Failed to connect best blocktor: Thread interrupt\r\n2016-11-11 14:15:09 torcontrol thread exit\r\n2016-11-11 14:15:09 net thread interrupt\r\n2016-11-11 14:15:09 addcon thread interrupt\r\n2016-11-11 14:15:09 opencon thread interrupt\r\n2016-11-11 14:15:09 scheduler thread interrupt\r\n2016-11-11 14:15:09 msghand thread interrupt\r\n2016-11-11 14:15:09 Shutdown: In progress...\r\n2016-11-11 14:15:10 StopNode()\r\n2016-11-11 14:15:10 IO error: C:\\Users\\PC\\AppData\\Roaming\\Bitcoin\\chainstate\\304129.ldb: Impossibile eseguire la richiesta a causa di un errore di dispositivo I/O.\r\n2016-11-11 14:15:10 *** System error while flushing: Database I/O error\r\n2016-11-11 14:15:15 Shutdown: done\r\n\r\n\r\nI'm worried because I have  bitcoin in this wallet.\r\nhow can I recover them?",
      "created_at" : "2016-11-11T14:31:33Z",
      "html_url" : "https://github.com/bitcoin/bitcoin/issues/9119#issuecomment-259968996",
      "id" : 259968996,
      "issue_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/9119",
      "updated_at" : "2016-11-11T14:31:33Z",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/259968996",
      "user" : {
         "avatar_url" : "https://avatars.githubusercontent.com/u/23367015?v=3",
         "events_url" : "https://api.github.com/users/Bitoma/events{/privacy}",
         "followers_url" : "https://api.github.com/users/Bitoma/followers",
         "following_url" : "https://api.github.com/users/Bitoma/following{/other_user}",
         "gists_url" : "https://api.github.com/users/Bitoma/gists{/gist_id}",
         "gravatar_id" : "",
         "html_url" : "https://github.com/Bitoma",
         "id" : 23367015,
         "login" : "Bitoma",
         "organizations_url" : "https://api.github.com/users/Bitoma/orgs",
         "received_events_url" : "https://api.github.com/users/Bitoma/received_events",
         "repos_url" : "https://api.github.com/users/Bitoma/repos",
         "site_admin" : false,
         "starred_url" : "https://api.github.com/users/Bitoma/starred{/owner}{/repo}",
         "subscriptions_url" : "https://api.github.com/users/Bitoma/subscriptions",
         "type" : "User",
         "url" : "https://api.github.com/users/Bitoma"
      }
   },
   {
      "body" : "I run a disk checks/diagnostics. Everything seems ok. But, when bitcoin core start again , after a while it stops. the log debug writes:\n\n2016-11-11 14:15:00 IO error: C:\\Users\\PC\\AppData\\Roaming\\Bitcoin\\chainstate\\304129.ldb: Impossibile eseguire la\nrichiesta a causa di un errore di dispositivo I/O.\n2016-11-11 14:15:00 *** System error while flushing: Database I/O error\n2016-11-11 14:15:09 Failed to connect best blocktor: Thread interrupt\n2016-11-11 14:15:09 torcontrol thread exit\n2016-11-11 14:15:09 net thread interrupt\n2016-11-11 14:15:09 addcon thread interrupt\n2016-11-11 14:15:09 opencon thread interrupt\n2016-11-11 14:15:09 scheduler thread interrupt\n2016-11-11 14:15:09 msghand thread interrupt\n2016-11-11 14:15:09 Shutdown: In progress...\n2016-11-11 14:15:10 StopNode()\n2016-11-11 14:15:10 IO error: C:\\Users\\PC\\AppData\\Roaming\\Bitcoin\\chainstate\\304129.ldb: Impossibile eseguire la richiesta a causa di un errore di dispositivo I/O.\n2016-11-11 14:15:10 *** System error while flushing: Database I/O error\n2016-11-11 14:15:15 Shutdown: done\n\nI'm worried because I have bitcoin in this wallet.\nhow can I recover them? can you help me?\n\n\n\n________________________________\nDa: Wladimir J. van der Laan <notifications@github.com>\nInviato: mercoledÃÂ¬ 9 novembre 2016 13.39\nA: bitcoin/bitcoin\nCc: Bitoma; Author\nOggetto: Re: [bitcoin/bitcoin] wallet 0.13.1 shutdown with debug record (#9119)\n\n\nYes, you could try running some disk checks/diagnostics. If the disk is broken the only safe course of action would be to replace it.\n\n-\nYou are receiving this because you authored the thread.\nReply to this email directly, view it on GitHub<https://github.com/bitcoin/bitcoin/issues/9119#issuecomment-259533965>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AWSNZ-8FjjAYeO0EtRKuhgSZxoZEu_alks5q8j2CgaJpZM4Kt_2j>.\n",
      "created_at" : "2016-11-11T15:00:13Z",
      "html_url" : "https://github.com/bitcoin/bitcoin/issues/9119#issuecomment-259975148",
      "id" : 259975148,
      "issue_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/9119",
      "updated_at" : "2016-11-11T15:00:13Z",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/259975148",
      "user" : {
         "avatar_url" : "https://avatars.githubusercontent.com/u/23367015?v=3",
         "events_url" : "https://api.github.com/users/Bitoma/events{/privacy}",
         "followers_url" : "https://api.github.com/users/Bitoma/followers",
         "following_url" : "https://api.github.com/users/Bitoma/following{/other_user}",
         "gists_url" : "https://api.github.com/users/Bitoma/gists{/gist_id}",
         "gravatar_id" : "",
         "html_url" : "https://github.com/Bitoma",
         "id" : 23367015,
         "login" : "Bitoma",
         "organizations_url" : "https://api.github.com/users/Bitoma/orgs",
         "received_events_url" : "https://api.github.com/users/Bitoma/received_events",
         "repos_url" : "https://api.github.com/users/Bitoma/repos",
         "site_admin" : false,
         "starred_url" : "https://api.github.com/users/Bitoma/starred{/owner}{/repo}",
         "subscriptions_url" : "https://api.github.com/users/Bitoma/subscriptions",
         "type" : "User",
         "url" : "https://api.github.com/users/Bitoma"
      }
   },
   {
      "body" : "@Bitoma Make a backup of your wallet.dat if you're worried about your bitcoins - this error refers to LevelDB, which is not the DB used for wallet.dat so probably no need to worry - you should backup your wallet.dat anyway as harddrives don't last forever.",
      "created_at" : "2016-11-13T03:22:57Z",
      "html_url" : "https://github.com/bitcoin/bitcoin/issues/9119#issuecomment-260164469",
      "id" : 260164469,
      "issue_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/9119",
      "updated_at" : "2016-11-13T03:22:57Z",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/260164469",
      "user" : {
         "avatar_url" : "https://avatars.githubusercontent.com/u/1530283?v=3",
         "events_url" : "https://api.github.com/users/rebroad/events{/privacy}",
         "followers_url" : "https://api.github.com/users/rebroad/followers",
         "following_url" : "https://api.github.com/users/rebroad/following{/other_user}",
         "gists_url" : "https://api.github.com/users/rebroad/gists{/gist_id}",
         "gravatar_id" : "",
         "html_url" : "https://github.com/rebroad",
         "id" : 1530283,
         "login" : "rebroad",
         "organizations_url" : "https://api.github.com/users/rebroad/orgs",
         "received_events_url" : "https://api.github.com/users/rebroad/received_events",
         "repos_url" : "https://api.github.com/users/rebroad/repos",
         "site_admin" : false,
         "starred_url" : "https://api.github.com/users/rebroad/starred{/owner}{/repo}",
         "subscriptions_url" : "https://api.github.com/users/rebroad/subscriptions",
         "type" : "User",
         "url" : "https://api.github.com/users/rebroad"
      }
   },
   {
      "body" : "Thanks, I was able to recover bitcoin",
      "created_at" : "2016-11-13T21:53:14Z",
      "html_url" : "https://github.com/bitcoin/bitcoin/issues/9119#issuecomment-260215687",
      "id" : 260215687,
      "issue_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/9119",
      "updated_at" : "2016-11-13T21:53:14Z",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/260215687",
      "user" : {
         "avatar_url" : "https://avatars.githubusercontent.com/u/23367015?v=3",
         "events_url" : "https://api.github.com/users/Bitoma/events{/privacy}",
         "followers_url" : "https://api.github.com/users/Bitoma/followers",
         "following_url" : "https://api.github.com/users/Bitoma/following{/other_user}",
         "gists_url" : "https://api.github.com/users/Bitoma/gists{/gist_id}",
         "gravatar_id" : "",
         "html_url" : "https://github.com/Bitoma",
         "id" : 23367015,
         "login" : "Bitoma",
         "organizations_url" : "https://api.github.com/users/Bitoma/orgs",
         "received_events_url" : "https://api.github.com/users/Bitoma/received_events",
         "repos_url" : "https://api.github.com/users/Bitoma/repos",
         "site_admin" : false,
         "starred_url" : "https://api.github.com/users/Bitoma/starred{/owner}{/repo}",
         "subscriptions_url" : "https://api.github.com/users/Bitoma/subscriptions",
         "type" : "User",
         "url" : "https://api.github.com/users/Bitoma"
      }
   },
   {
      "body" : "Ã¢ÂÂGood to hear. Make sure to backup your wallet.dat regularly in the future\nas well. Also, feel free to close this issue if everything is resolved.\n",
      "created_at" : "2016-11-13T22:06:39Z",
      "html_url" : "https://github.com/bitcoin/bitcoin/issues/9119#issuecomment-260216477",
      "id" : 260216477,
      "issue_url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/9119",
      "updated_at" : "2016-11-13T22:06:39Z",
      "url" : "https://api.github.com/repos/bitcoin/bitcoin/issues/comments/260216477",
      "user" : {
         "avatar_url" : "https://avatars.githubusercontent.com/u/6399679?v=3",
         "events_url" : "https://api.github.com/users/MarcoFalke/events{/privacy}",
         "followers_url" : "https://api.github.com/users/MarcoFalke/followers",
         "following_url" : "https://api.github.com/users/MarcoFalke/following{/other_user}",
         "gists_url" : "https://api.github.com/users/MarcoFalke/gists{/gist_id}",
         "gravatar_id" : "",
         "html_url" : "https://github.com/MarcoFalke",
         "id" : 6399679,
         "login" : "MarcoFalke",
         "organizations_url" : "https://api.github.com/users/MarcoFalke/orgs",
         "received_events_url" : "https://api.github.com/users/MarcoFalke/received_events",
         "repos_url" : "https://api.github.com/users/MarcoFalke/repos",
         "site_admin" : false,
         "starred_url" : "https://api.github.com/users/MarcoFalke/starred{/owner}{/repo}",
         "subscriptions_url" : "https://api.github.com/users/MarcoFalke/subscriptions",
         "type" : "User",
         "url" : "https://api.github.com/users/MarcoFalke"
      }
   }
]
